sdk/keyvault/azure-security-keyvault/azure/security/keyvault/certificates/_client.py

```diff
@@ -0,0 +1,222 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import Any, Dict, Mapping, Optional
+from datetime import datetime
+
+from azure.core.pipeline import Pipeline
+from azure.core.pipeline.transport import RequestsTransport
+from azure.security.keyvault._internal import _BearerTokenCredentialPolicy
+
+from .._generated import KeyVaultClient
+from ._models import (
+    Certificate,
+    CertificateBase,
+    DeletedCertificate,
+    CertificatePolicy,
+    Issuer,
+    IssuerBase,
+    Contact,
+    CertificateOperation,
+)
+
+
+class CertificateClient:
+    @staticmethod
+    def create_config(**kwargs):
+        pass  # TODO
+
+    def __init__(self, vault_url, credentials, config=None, api_version=None, **kwargs):
+        if not credentials:
+            raise ValueError("credentials")
+
+        if not vault_url:
+            raise ValueError("vault_url")
+
+        self._vault_url = vault_url
+
+        if api_version is None:
```

Comment:
Is there a specific reason we are not providing `KeyVaultClient.DEFAULT_API_VERSION` as the default value for the parameter?

Reply:
No. I've done this in a base class init for all the sub-clients, in another branch which will land before we want to merge a certs implementation.